test/manipulation.test.js

```diff
@@ -934,6 +956,23 @@ describe('manipulation', function() {
         });
       });
 
+      it('should have updated password hashed with replaceAttributes',
+      function(done) {
+        StubUser.create({ password: 'foo' }, function(err, created) {
+          if (err) return done(err);
+          created.replaceAttributes({ password: 'test' }, { validate: false },
```

Comment:
Let's fix the other problem first then and keep this patch clean of tweaks/hacks that are difficult to understand.